docs/sql-ref-syntax-aux-describe-table.md

```diff
@@ -173,6 +236,10 @@ DESCRIBE customer salesdb.customer.name;
 |data_type|    string|
 |  comment|Short name|
 +---------+----------+
+
+-- Returns the table metadata in JSON format.
+DESC FORMATTED customer AS JSON;
+{"table_name":"customer","catalog_name":"spark_catalog","namespaces":["default"],"columns":[{"name":"cust_id","type":{"type":"integer"}},{"name":"name","type":{"type":"string"},"comment":"Short name"},{"name":"state","type":{"type":"varchar(20)"}}],"location": "file:/tmp/salesdb.db/custom...","created_time":"2020-04-07T14:05:43Z","last_access":"UNKNOWN","created_by":"None","type":"MANAGED","provider":"parquet","partition_provider":"Catalog","partition_columns":["state"]}
```

Comment:
Didn't we want to shred the types? I.e. VARCHAR is the type and 20 is the length.
Similarly for DECIMAL we would have precision and scale. MAP and STRUCT need to be nested....

Reply:
thanks for catching, this was outdated -- I re-ran and updated with the correct columns format:

```
"columns":[{"name":"cust_id","type":{"type":"integer"},"nullable":true},{"name":"name","type":{"type":"string"},"nullable":true,"comment":"Short name","metadata":{"comment":"Short name"}},{"name":"state","type":{"type":"varchar","length":20},"nullable":true}]
```